content/utilities/you-snooze-you-lose/plugin.js

```diff
@@ -0,0 +1,278 @@
+        // Get moves stats
+        let totalMoves = 0;    // after loop, should equal this.arrivals.length
+        let totalMovesSq = 0;  // needs loop to calculate
+        for (let i = 0; i < HOURS_IN_DAY; i++) {
+          const mv = this.movesResults[i] || 0;
+          totalMoves += mv;
+          totalMovesSq += mv * mv;
+        }
+    
+        // If moves found, redraw the table with colour bands
+        this.resetResultsTable();
+        if (totalMoves < 1) {
+          this.resultsTable.append(getResultsDiv(`No moves found in last ${HOURS_IN_DAY} hours`, COL_ERR));
+        } else {
+          this.resultsTable.append(getResultsDiv(`Total moves found: ${totalMoves}`, COL_TOTAL));
+          const avg = totalMoves / HOURS_IN_DAY;  // mean average
+          const sd = (totalMovesSq / HOURS_IN_DAY - avg * avg) ** 0.5;  // standard deviation
+          for (let i=0; i<HOURS_IN_DAY; i++) {
+            const subtractHour = i + 1;
+            const numberOfMoves = this.movesResults[i] || 0;
+            const ss = sd > 0 ? (numberOfMoves - avg) / sd : 0;  // standard score
+
+            // Construct index from 0 (few moves) to 4 (more than average moves) for the colour
+            const idx = ss < SS0 ? 0 : ss < SS1 ? 1 : ss < SS2 ? 2 : ss < SS3 ? 3 : 4;
+            const bgCol = COL_SCORES[idx];
+            this.resultsTable.append(getResultsDiv(
+              `${subtractHour-1} to ${subtractHour} hours ago: ${numberOfMoves} move${numberOfMoves===1?'':'s'}`,
+              bgCol,
+              RESULTS_SIZE
+            ));
+          }
+        }
+      }
+    }
+
+    // Async search finished (with success or failure), reset status so it can run again
+    this.searching = false;
+  }
+
+  async render(container) {
+    container.style.textAlign = 'center';
+    container.style.width = WINDOW_WIDTH;
+    container.style.height = WINDOW_HEIGHT;
+    if (DEV_MODE) container.style.backgroundColor = '#000088';
+
+    // Input row
+    container.appendChild(this.userInput);
+    const clearButton = document.createElement('button');
+    clearButton.innerText = 'clear';
+    clearButton.onclick = () => {
+      this.userInput.value = '';
+    };
+    clearButton.style.marginLeft = '5px';
+    clearButton.style.width = BUTTON_CLEAR_WIDTH;
+    clearButton.style.height = UI_ELT_SMALL_HEIGHT;
+    container.appendChild(clearButton);
+
+    // Search button
+    const searchButton = document.createElement('button');
+    searchButton.innerText = 'Check for snoozing!';
+    searchButton.onclick = this.doSearch.bind(this);
+    searchButton.style.width = BUTTON_SEARCH_WIDTH;
+    searchButton.style.height = UI_ELT_HEIGHT;
+    const searchRow = document.createElement('div');
+    searchRow.style.margin = '15px 0px 10px 0px';
+    searchRow.appendChild(searchButton);
+    container.appendChild(searchRow);
+
+    // Results table
+    container.appendChild(this.resultsTable);
+  }
+
+  destroy() {
+  }
+}
+
+export default YouSnoozeYouLose;
```

Comment:
```suggestion
export default YouSnoozeYouLose;

```
nitpick Github wants a newline ;)